coalib/bearlib/aspects/taste.py

```diff
@@ -87,5 +87,5 @@ def __set__(self, obj, value):
         """
         if self.name in obj.__dict__:
             raise AttributeError(
-                "can't set taste values of aspectclass instances")
+                "A 'taste' value of aspectclass instance exists already.")
```

Comment:
As @NiklasMM pointed out, please change the message to `A 'taste' value for this aspectclass instance exists already` here and also in the tests. :)